lxd/images.go

```diff
@@ -147,7 +147,7 @@ func compressFile(compress string, infile io.Reader, outfile io.Writer) error {
 			return fmt.Errorf("tar2sqfs: %v (%v)", err, strings.TrimSpace(string(output)))
 		}
 		// Replay the result to outfile
-		tempfile.Seek(0, 0)
+		_, _ = tempfile.Seek(0, 0)
```

Comment:
I think this error should be handled and returned as if seeking fails then the function has failed.